### PR TITLE
#17687  postman and minor improvements

### PR DESCRIPTION
**Pull request by fabrizzio-dotCMS**
This PR introduces the postman test for the new integration-services resource.

### CODE REVIEW

**Review by jdotcms — APPROVED** [review]

File: dotCMS/src/main/java/com/dotcms/rest/api/v1/secret/ServiceIntegrationHelper.java [hunk 1]
```diff
@@ -324,23 +325,40 @@ private void validateIncomingParams(final Map<String, Param> incomingParams, fin
         for (final Entry<String, Param> incomingParamEntry : incomingParams.entrySet()) {
             final String incomingParamName = incomingParamEntry.getKey();
             final Param describedParam = serviceDescriptorParams.get(incomingParamName);
+            final Param incomingParam = incomingParamEntry.getValue();
```
Comment (dev-dotcms) [inline dotCMS/src/main/java/com/dotcms/rest/api/v1/secret/ServiceIntegrationHelper.java]: Codacy Issue found: [Avoid declaring a variable if it is unreferenced before a possible exit point.](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=4979325)

File: dotCMS/src/main/java/com/dotcms/rest/api/v1/secret/ServiceIntegrationHelper.java [hunk 2]
```diff
@@ -324,23 +325,40 @@ private void validateIncomingParams(final Map<String, Param> incomingParams, fin
...
             if(describedParam.isRequired() && UtilMethods.isNotSet(incomingParam.getValue())){
                throw new IllegalArgumentException(
                String.format("Params named `%s` is marked as required in the descriptor but does not have any value.", incomingParamName));
             }
+
+
+
+            if(Type.BOOL.equals(describedParam.getType())){
+                if(UtilMethods.isSet(incomingParam.getValue())){
+                    if(!("true".equalsIgnoreCase(incomingParam.getValue()) || "false".equalsIgnoreCase(incomingParam.getValue()))){
```
Comment (dev-dotcms) [inline dotCMS/src/main/java/com/dotcms/rest/api/v1/secret/ServiceIntegrationHelper.java]: Codacy Issue found: [These nested if statements could be combined](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=4979325)

File: dotCMS/src/main/java/com/dotcms/rest/api/v1/secret/ServiceIntegrationHelper.java [hunk 3]
```diff
@@ -324,23 +325,40 @@ private void validateIncomingParams(final Map<String, Param> incomingParams, fin
...
             //We revise the incoming param against the definition loaded from the yml.
             if(describedParam.isRequired() && UtilMethods.isNotSet(incomingParam.getValue())){
                throw new IllegalArgumentException(
                String.format("Params named `%s` is marked as required in the descriptor but does not have any value.", incomingParamName));
             }
+
+
+
+            if(Type.BOOL.equals(describedParam.getType())){
+                if(UtilMethods.isSet(incomingParam.getValue())){
```
Comment (dev-dotcms) [inline dotCMS/src/main/java/com/dotcms/rest/api/v1/secret/ServiceIntegrationHelper.java]: Codacy Issue found: [These nested if statements could be combined](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=4979325)

File: dotCMS/src/main/java/com/dotcms/rest/api/v1/secret/ServiceIntegrationHelper.java [hunk 4]
```diff
@@ -380,20 +398,24 @@ private void validateIncomingParams(final Set<String> incomingParamNames, final
...
      * @throws DotDataException
      */
-    void createServiceIntegration(final FormDataMultiPart multipart, final User user)
+    List<ServiceIntegrationView> createServiceIntegration(final FormDataMultiPart multipart, final User user)
             throws IOException, DotDataException {
         final List<File> files = new MultiPartUtils().getBinariesFromMultipart(multipart);
         if(!UtilMethods.isSet(files)){
             throw new DotDataException("Unable to extract any files from multi-part request.");
         }
+        List<ServiceIntegrationView> serviceIntegrationViews = new ArrayList<>(files.size());
```
Comment (dev-dotcms) [inline dotCMS/src/main/java/com/dotcms/rest/api/v1/secret/ServiceIntegrationHelper.java]: Codacy Issue found: [Local variable 'serviceIntegrationViews' could be declared final](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=4979325)

File: dotCMS/src/main/java/com/dotcms/rest/api/v1/secret/ServiceIntegrationHelper.java [hunk 5]
```diff
@@ -380,20 +380,24 @@ private void validateIncomingParams(final Set<String> incomingParamNames, final
...
```
Comment (dev-dotcms) [inline dotCMS/src/main/java/com/dotcms/rest/api/v1/secret/ServiceIntegrationHelper.java]: Codacy Issue found: [Local variable 'serviceIntegrationViews' could be declared final](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=4979325)

**Review by dsilvam — APPROVED** [review]

**Review by jgambarios — APPROVED** [review]

**Pull request merged**